src/components/course/course-header/CourseRunCards.jsx

```diff
@@ -28,18 +29,20 @@ const CourseRunCards = () => {
       hasEqualColumnHeights={false}
     >
       {availableCourseRuns.map((courseRun) => {
-        const redeemabilityForContentKey = redeemabilityPerContentKey?.find(r => r.contentKey === courseRun.key);
-        const redeemableSubsidyAccessPolicy = redeemabilityForContentKey?.redeemableSubsidyAccessPolicy;
+        const hasRedeemablePolicy = userSubsidyApplicableToCourse?.subsidyType === LEARNER_CREDIT_SUBSIDY_TYPE;
```

Comment:
[inform] By refactoring to use `userSubsidyApplicableToCourse`, we ensure the priority order of subsidy types is already taken into account, i.e. if a subscription license is applicable to the course, `userSubsidyApplicableToCourse` would represent a subsidy type of `license` instead of `learnerCredit`.